app/views/coronavirus_pages/shared/_github.html.erb

```diff
@@ -1,18 +1,18 @@
 <div class="govuk-!-padding-bottom-1">
   <%= render "govuk_publishing_components/components/heading", {
-  text: "1. Update GitHub",
+  text: t("coronavirus_pages.github.heading"),
```

Comment:
I imagine if we're having some headings numbered it'd be good to have them all done. Otherwise the suffix of a 2 is a bit of a surprise.

But, potentially better than numbers could be to have contextual names for example: `coronavirus_pages.github.update_heading` and `coronavirus_pages.github.review_heading` ?

Reply:
Thanks! Have updated.